src/Components/Custom-Button/index.js

```diff
@@ -1,4 +1,20 @@
 import React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+const button = styled.button`
+  color: #aa3939;
```

Comment:
- [ ] Let's allow for the color to be changed. To do this we could change this line to `color: ${props => color ? color : #aa3939}`